src/components/LoadingScreen.tsx

```diff
@@ -139,15 +132,18 @@ class LoadingScreen extends Component {
       sendReq.pipe(str).pipe(file)
     }) // close() is async, call cb after close completes
 
-    file.on('finish', () => file.close(cb)) // check for request errors
+    file.on('finish', () => {
+      file.close()
+      cb()
+    }) // check for request errors
 
     sendReq.on('error', err => {
-      fs.unlink(dest)
+      fs.promises.unlink(dest)
```

Comment:
I think changing this is not safe. I woudn't change `fs.unlink` to `fs.promises.unlink`.